dali/operators/generic/slice/slice_attr.cc

```diff
@@ -22,19 +22,30 @@ namespace dali {
 DALI_SCHEMA(SliceAttr)
     .DocStr(R"code(Slice attributes placeholder)code")
     .AddOptionalArg("axes",
-        R"code(Order of dimensions used for anchor and shape slice inputs, as dimension indexes)code",
+        R"code(Order of dimensions used for the anchor and shape slice inputs as dimension
+indexes.)code",
         std::vector<int>{1, 0})
     .AddOptionalArg("axis_names",
-        R"code(Order of dimensions used for anchor and shape slice inputs, as described in layout.
-If provided, `axis_names` takes higher priority than `axes`)code",
+        R"code(Order of the dimensions used for the anchor and shape slice inputs,
+as described in layout.
+
+If a value is provided, ``axis_names`` will have a higher priority than ``axes``.)code",
         TensorLayout("WH"))
     .AddOptionalArg("normalized_anchor",
-        R"code(Whether or not the `anchor` input should be interpreted as normalized (range [0.0, 1.0])
-or absolute coordinates)code",
+        R"code(Determines whether the anchor input should be interpreted as normalized
+(range [0.0, 1.0]) or as absolute coordinates.
+
+.. note::
+    This argument is only relevant when anchor input is a float tensor. For integer types,
```

Comment:
"float tensor" doesn't sound well.
```suggestion
    This argument is only relevant when anchor data type is ``float``. For integer types,
```
?